packages/core/src/indexing/postgres/migrations/1-create-model-table.ts

```diff
@@ -34,5 +36,8 @@ export async function createModelTable(dataSource: Knex, tableName: string) {
         storageEngineIndexType: 'hash',
       }
     )
+    table.index(['stream_id', 'tip'], `idx_${indexName}_stream_id_tip`, {
```

Comment:
do we need this index?  how would it be used?

Reply:
ooh, yes I thought wrongly inserts vs. updates (override) here. I'll remove the index.